opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -146,7 +145,7 @@ impl Drop for LoggerProviderInner {
     fn drop(&mut self) {
         for processor in &mut self.processors {
             if let Err(err) = processor.shutdown() {
-                global::handle_error(err);
+                otel_error!(name: "LoggerProviderInner.Drop",  otel_name = "LoggerProviderInner.Drop", error = format!("{:?}", err));
```

Comment:
any warning/error level message must be user understandable and should not refer to internal constructs like 'Inner'.
name:LoggerProvideShutdownOnDrop
msg = "LoggerProvider shutdown() failed when invoked from Drop"
error = "error"

Its okay to handle this in a follow up - so this PR can focus on replacing global error handler with internal logs.

Reply:
We can have name to follow pattern like:
name  = `<Module>.<Funtion>.<Action>`
So in this case it would be `LoggerProvider.Drop.Error` or `LoggerProvider.Drop.ShutdownError`. The actual error message would be contained in the error `attribute`. And if required, more attributes can be there for specific field values.

Reply:
module.func.action pattern is good pattern to follow, for debug logging that owners of this crate use to debug. I don't think we can have such a pattern for user-actionable ones, as they don't need to know which module/component is triggering this error.